src/applications/disability-benefits/all-claims/tests/utils/utils.unit.spec.jsx

```diff
@@ -560,6 +595,44 @@ describe('isAnswering781Questions', () => {
     };
     expect(isAnswering781Questions(1)(formData)).to.be.false;
   });
+
+  describe('PTSD flow migration', () => {
+    const ptsdFormData = {
+      newDisabilities: [
+        {
+          condition: 'Ptsd personal trauma',
+        },
+      ],
+      'view:selectablePtsdTypes': {
+        'view:combatPtsdType': true,
+      },
+      'view:upload781Choice': 'answerQuestions',
+    };
+
+    describe('when the modern 0781 flow is enabled', () => {
+      const ptsdFormDataModernFlowEnabled = {
+        syncModern0781Flow: true,
+        ...ptsdFormData,
```

Comment:
Wouldn't it not be possible to set values for `combatPtsdType` or `upload781Choice` if `syncModern0781Flow` is enabled?

Reply:
See https://github.com/department-of-veterans-affairs/vets-website/pull/34515#discussion_r1968405414